src/SprykerEco/Client/Easycredit/Dependency/Client/EasycreditToZedRequestClientBridge.php

```diff
@@ -27,7 +27,7 @@ public function __construct($zedRequestClient)
     /**
      * @param string $url
      * @param \Spryker\Shared\Kernel\Transfer\TransferInterface $object
-     * @param array|int|null $requestOptions
+     * @param array<mixed>|int|null $requestOptions
```

Comment:
According to this doc https://spryker.atlassian.net/wiki/spaces/CORE/pages/1395687852/CS+-+Docblocks+and+Collections you need to define key type too, e.g. array<string, mixed>